app/src/sharedTest/java/org/oppia/app/RecyclerViewMatcher.kt

```diff
@@ -0,0 +1,54 @@
+package org.oppia.app
+
+import android.content.res.Resources
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+class RecyclerViewMatcher(private val recyclerViewId: Int) {
```

Comment:
Also, do we need the wrapper RecyclerViewMatcher? Could we just have a top-level Kotlin file with the atPosition method and import it where needed? The BindableAdapterTest version lends itself to this approach since it follows the standard Espresso matcher idiom of being a static-esque, standalone method.

Reply:
Removed wrapper and started using `atPosition` and `atPositionView` method directly.